scripts/sonic_py.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+
+"""
+    Sonic_py is the library that provides APIs to access database.
+"""
+
+import redis
+
+
+REDIS_UNIX_SOCKET = "/var/run/redis/redis.sock"
+CNT_DB = 2
+COUNTER_PORT_NAME_MAP = "COUNTERS_PORT_NAME_MAP"
+
+class sonic_py(object):
```

Comment:
https://github.com/Azure/sonic-py-swsssdk - Python utility library for sonic/switch state service database access. Should we use them?